sensor/kubernetes/orchestratornamespaces/store.go

```diff
@@ -12,6 +12,15 @@ type OrchestratorNamespaces struct {
 	lock  sync.RWMutex
 }
 
+// ReconcileDelete is called after Sensor reconnects with Central and receives its state hashes.
+// Reconciliacion ensures that Sensor and Central have the same state by checking whether a given resource
+// shall be deleted from Central.
+func (n *OrchestratorNamespaces) ReconcileDelete(resType, resID string, resHash uint64) (string, error) {
+	_, _, _ = resType, resID, resHash
+	// TODO implement me
+	panic("implement me")
```

Comment:
Let's be super sure this isn't being called at all at the state of this PR to avoid panics in master